tools/data_processing_v2_beta/processors/daq_processing.py

```
@@ -78,117 +78,77 @@ def __init__(
-        # TODO: Make this better (dynamic column creation, select which cols, etc.)
-        with open(file_path, "w") as outfile:
-            writer = csv.writer(outfile)
-            writer.writerow(["Timestamp (ns) +- 10ns"] + self.processed_data[0].sensors)
-            for datapoint in self.processed_data:
-                writer.writerow([datapoint.ni_relative_timestamp] + datapoint.values)
-            export_size = os.path.getsize(file_path)
-            formatted_can_size = "{:.2f} MB".format(export_size / (1024 * 1024))
-        return formatted_can_size
+                
+                # Verify that the unpacked_data is actually valid
+                for key in DAQ_EXPECTED_RECEIVE_DATA_FORMAT.keys():
```

Ruff (0.8.2)

Use `key in dict` instead of `key in dict.keys()`

Remove `.keys()`

(SIM118)